.gitignore

```diff
@@ -1,3 +1,5 @@
 main
 dist/*
-packer-plugin-alicloud
\ No newline at end of file
+packer-plugin-alicloud
+/.idea/
```

Comment:
This looks redundant with the `.idea` below, is it voluntary?

Reply:
./idea/ is deleted in the latest commit